doltpy_integrations/metaflow/examples/hospital_bounty/blog_post.md

```diff
@@ -0,0 +1,170 @@
+
+## Setup
+Let's get the boring stuff out of the way. We need the following:
+- Dolt and `doltpy-integrations` installed
+- Metaflow installed
+- the sample dataset we will use, which can easily clone from DoltHub
+
+### Install Dolt
+The first step is to install Dolt on a `*nix` system:
+```
+sudo bash -c 'curl -L https://github.com/dolthub/dolt/releases/latest/download/install.sh | bash'
+```
+
+There are Windows distributions and a Homebrew cask. Find more details about installation [here](https://docs.dolthub.com/getting-started/installation).
+
+### install doltpy-integrations[metaflow]
+Next let's install the Metaflow + Dolt integration. It comes packaged with both Metaflow, and Dolt's Python API, Doltpy. It's easy enough to install via `pip`:
+```
+pip install doltpy-integraions[metaflow]
+```
+
+### Get The Data
+The final step is to acquire the dataset. Dolt is a SQL database with Git-like version control features. That includes the ability to clone a remote to your local machine. We do that here:
+```
+$ dolt clone dolthub/hospital-price-transparency && cd hospital-price-transparency
+
+```
+
+Note this dataset is quite large, and could take a few minutes to clone. Once it's landed it's straightforward to jump write into SQL:
+```
+$ dolt sql
+...
+```
+
+## Arm Chair Data Science
+We are going to demonstrate a sample flow defined in Metaflow that is designed to a hypothesis about our sample dataset. The data set we acquired is the result of a "data bounty." A data bounty is crowd sourced data gathering project sponsored by DoltHub. You can read more about this particular bounty in this blog [post](https://www.dolthub.com/blog/2021-03-03-hpt-bounty-review/).
+
+The hyposthesis we wish to test is this:
+> As the observation count increases, the procedure level price variance should decrease
+
+Let's jump into defining a Metaflow pipeline that will answer this question.
+
+## Using Metaflow
+In this case we are using Dolt as the input, as well as for storing intermediate results and outputs. In many use-cases the upstream datasource might come from a data warehouse of some kind. The architecture for our flow looks something like:
+
+
+Our flow has three steps:
+- `start`: compute and store state level medians for every procedure
+- `variances`: compute procedure level variances across states
+- `end`: required by Metaflow
+
+In this case `start` reads and computes an intermediate result. This result is stored to Dolt. `variances` uses this intermediate result to compute the data of interest. The state level medians are not especially taxing to compute, but in a real world setting it's perfectly plausible that an intermediate result is used by many downstream jobs and cannot ergonomically be recomputed on the fly. Versioning provides read isolation to users of that intermediate result. This what the flow looks like schematically:
```

Comment:
I really like this paragraph.

Reply:
I ended up dropping it, but I might work back in some how as I agree it explains something cool